filters/reorient.cpp

```diff
@@ -113,6 +113,7 @@ tesseract_version_before_(const char *cutoff)
           log::debug ("found tesseract-%1%") % result;
 
           return (0 > strverscmp (result, cutoff));
+          return (false);
```

Comment:
This will never be reached but you already fixed that in a later commit.  In such a case you can rebase interactively and merge the two commits into one.  That way, I won't be scratching my head for a while :wink: